static/src/stylesheets/module/content-garnett/_article-immersive.scss

```diff
@@ -714,14 +740,23 @@
         }
     }
     .drop-cap {
-        height: 7.25em;
+        height: 105px;
 
         .drop-cap__inner {
             font-weight: 900;
-            font-size: 9.5em;
+            font-size: 140px;
             line-height: .7; // To optically align to top of five lines of text
+
+            @include mq($from: tablet) {
+                font-size: 138px;
```

Comment:
I tried the promising [`initial-letter`](https://developer.mozilla.org/en-US/docs/Web/CSS/initial-letter) but alas it is not available in many browsers yet 😩